### DISCUSSION TITLE
MRR Questions

### DISCUSSION THREAD

**Discussion by saki2fifty** [RainbowMiner/RainbowMiner#1690] [Q&A] [answered]
How can we disable MRR altogether after the current rental has finished, without disrupting the rental?  PauseBetweenRentals then EnableAutoUpdate?

What is the purpose of mrr.config.txt?  My settings currently reside in pools.config.txt.  Didn't see it mentioned in the docs.

If I make a change to say EnableAutoExtend, while it's currently rented, will RBM pick up this change to extend the current rental?

Any plans on adding MRR config settings to the web console?

**Comment by RainbowMiner** [comment, marked as answer]
> How can we disable MRR altogether after the current rental has finished, without disrupting the rental? PauseBetweenRentals then EnableAutoUpdate?

Yes, make the "PauseBetweenRentals" very very large. E.g `"PauseBetweenRentals": "3650 day",` - now it's waiting 10 years until the rigs get re-enabled. To completely disable MRR after the last rental has ended, just remove MiningRigRentals from "PoolName" in config.txt

> What is the purpose of mrr.config.txt? My settings currently reside in pools.config.txt. Didn't see it mentioned in the docs.

The settings in mrr.config.txt are per worker name. In a default setup, there would be two workers. Your `"WorkerName"`, like defined in `config.txt` and a second, that has "cpu" added to the end. E.g. if your WorkerName would be "rainbow", you will find two setting instances in `mrr.config.txt`. One for "rainbow", the other for "rainbowcpu". 
You may also set another WorkerName for each device group in `devices.config.txt`. Another instance for each individual WorkerName will be created in `mrr.config.txt`
Settings in `mrr.config.txt` supersede the settings in pools.config.txt.

> If I make a change to say EnableAutoExtend, while it's currently rented, will RBM pick up this change to extend the current rental?

Yes - all settings will be reloaded, as soon as the config files have changed.

> Any plans on adding MRR config settings to the web console?

Yes. There will be a pool setting page.
